interface/wx/dc.h

```diff
@@ -1550,6 +1550,14 @@ class wxDC : public wxObject
     */
     wxSize GetPPI() const;
 
+    /**
+       Get the DPI scale factor. Can be used to scale coordinates and sizes
+       appropriate to the DPI of the associated wxWindow or wxDC.
+
+        @since 3.1.7
```

Comment:
Incredible minor, but there seems to be an extra space here (also below).

We probably should also use `@see wxWindow::GetScaleFactor()` which explains things in more details.